src/librbd/RefCountedRequest.h

```diff
@@ -0,0 +1,70 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
Nit: probably can just re-use the existing ```common/RefCountedObj``` instead of creating a new class

Reply:
The idea is to remove `image_ctx` from this class later (lighter version of reference counting).

Reply:
The ```RefCountedObj``` only holds a counter and ```cct``` pointer so it's already pretty light-weight.